sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs.csproj

```diff
@@ -6,7 +6,6 @@
     <Version>5.3.0-beta.1</Version>
     <!--The ApiCompatVersion is managed automatically and should not generally be modified manually.-->
     <ApiCompatVersion>5.2.0</ApiCompatVersion>
```

Comment:
Technically, if I clear this, I bypass API checks, no?

Reply:
Yes, but we already have the comment saying that it is managed automatically.